app/internal/email.py

```diff
@@ -9,14 +9,15 @@
 
 from app.config import (CALENDAR_HOME_PAGE, CALENDAR_REGISTRATION_PAGE,
                         CALENDAR_SITE_NAME, email_conf, templates)
-from app.database.models import Event, User
-
+from app.database.models import Event, User, UserEvent
+from app.internal.utils import get_current_user
 mail = FastMail(email_conf)
 
 
 def send(
         session: Session, event_used: int, user_to_send: int,
-        title: str, background_tasks: BackgroundTasks = BackgroundTasks
+        title: str, content: str = "",
+        background_tasks: BackgroundTasks = BackgroundTasks
```

Comment:
Add `,` (use precommit hooks to add this kind of stuff automatically)

Reply:
added precommit hooks :)